index.rst

```diff
@@ -1,13 +1,15 @@
 Welcome to the UBports documentation!
 =====================================
 
-Welcome to the official documentation of the UBports project!
+Welcome to the official documentation of the UBports project! We are a team of engineers, designers, community managers, translators, students and volunteers who together bring to life Ubuntu Touch, a mobile operating system focused on ease of use, privacy, and convergence. We are also lovers of Free software and open collaboration. You can find us all over the web, but Github provides the best starting point for planning your involvement. If you are new, head to get.ubuntu.com to get started
 
-UBports develops the mobile phone operating system Ubuntu Touch. Ubuntu Touch is a mobile operating system focused on ease of use, privacy, and convergence.
+Ubuntu Touch is a mobile operating system focused on ease of use, privacy, and convergence. It features a tablet-oriented user interface with access to a desktop interface when a keyboard or mouse is connected.
```

Comment:
It features a touch-oriented...

Isn't it about the combination of touch and mouse/keyboard?
Since most devices are phones, I personally would not call UT tablet-oriented.